src/framework/mlt.vers

```diff
@@ -624,3 +624,9 @@ MLT_7.1.0 {
     mlt_properties_get_properties;
     mlt_properties_get_properties_at;
 } MLT_7.0.0;
+
+MLT_7.4.0 {
```

Comment:
Why 7.4 instead of 7.2?

Reply:
7.1 was wrong. The symbol version is supposed to match the release version. That is not required, but that is this project's convention ever since 6.0.

Reply:
Or maybe you missed the fact that there was already a 7.2 release.

Reply:
Oh Yes, that explains it. And you also skip odd numbers for some reason.

Reply:
Odd version numbers is how we see whether someone is using a release or some interim version and need to get a git sha.